xbmc/addons/binary-addons/BinaryAddonManager.h

```diff
@@ -36,15 +40,31 @@ namespace ADDON
   typedef std::shared_ptr<CBinaryAddonBase> BinaryAddonBasePtr;
   typedef std::vector<BinaryAddonBasePtr> BinaryAddonBaseList;
 
-  class CBinaryAddonManager
+  class IBinaryAddonManagerCallback
```

Comment:
methods should be prefixed with On to make it clear they're events, also matches regular addons.